### PR TITLE
Release 1.3-next.0

### PR DESCRIPTION
**Pull request by cesarenaldi**

### CODE REVIEW

File: packages/client/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+### Minor Changes
+
+- c2a91ef4: **Added** `Profile.invitedBy` field
+- 4c4505d2: **Added** support for Profile Guardian
+
+### Patch Changes
+
+- 0f75f79d: **Added** experimental `useCurrentSession` hook
+  **Fixed** issue with `Profile` entity being leaked by the `useWalletLogin` hook
+  **Fixed** bug preventing logout on expired credentials detected at startup type
```
Comment (krzysu) [inline packages/client/CHANGELOG.md:14]: what do we do with these entries? they are not related to client SDK, might be the result of changes in underlying packages, but I don't think so in this case

File: packages/wagmi/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lens-protocol/wagmi",
-  "version": "2.0.1",
+  "version": "2.0.0-next.0",
```
Comment (krzysu) [inline packages/wagmi/package.json:3]: how did it go from 2.0.1 to 2.0.0-next?

**Review by desfero — APPROVED** [review]

**Review by krzysu — APPROVED** [review]

File: packages/client/CHANGELOG.md (hunk 1 above)
Comment (cesarenaldi) [reply packages/client/CHANGELOG.md]: Good catch, manually fixed.

File: packages/wagmi/package.json (hunk 2 above)
Comment (cesarenaldi) [reply packages/wagmi/package.json]: My bad. Fixed.

**Pull request merged**